src/Microsoft.AspNet.Mvc.Core/Rendering/HtmlHelperDisplayExtensions.cs

```diff
@@ -71,11 +71,14 @@ public static HtmlString Display([NotNull] this IHtmlHelper html, string express
         /// </para>
         /// </remarks>
         public static HtmlString Display(
-            [NotNull] this IHtmlHelper html,
+            [NotNull] this IHtmlHelper htmlHelper,
             string expression,
             object additionalViewData)
         {
-            return html.Display(expression, templateName: null, htmlFieldName: null,
+            return htmlHelper.Display(
+                expression,
+                templateName: null,
+                htmlFieldName: null,
```

Comment:
Based on other confusions mentioned above, just making a note here in-case we decide to change the naming.